pkg/virt-controller/watch/vm/vm.go

```diff
@@ -3004,8 +2842,15 @@ func (c *Controller) syncVMAnnotationsToVMI(vm *virtv1.VirtualMachine, vmi *virt
 		descheduler.EvictPodAnnotationKeyAlphaPreferNoEviction,
 	}
 
+	removedDeprecated := false
```

Comment:
nit: if super-lazy you could reuse `changed` for both for-loops that are essentially checking the changed condition